linera-client/src/wallet.rs

```diff
@@ -165,6 +167,7 @@ pub struct UserChain {
     pub timestamp: Timestamp,
     pub next_block_height: BlockHeight,
     pub pending_proposal: Option<PendingProposal>,
+    pub epoch: Option<Epoch>,
```

Comment:
Maybe this could be non-optional: For root chains, it starts at 0, for child chains it starts at the same epoch as the parent chain. Basically we should always get the epoch from the same source as we get the timestamp.